### ISSUE TITLE
Successfully parses invalid colon expression

### ISSUE THREAD

**Issue by domluna** [julia-vscode/CSTParser.jl#104]
```julia
julia> CSTParser.parse("1:2.^3:10")
ColonOpCall  9(9)
 INTEGER: 1  1(1)
 OP: COLON  1(1)
 BinaryOpCall  4(4)
  INTEGER: 2  1(1)
  OP: CIRCUMFLEX_ACCENT  2(2)
  INTEGER: 3  1(1)
 OP: COLON  1(1)
 INTEGER: 10  2(2)
```

vs.

```julia
julia> Meta.parse("1:2.^3:10")
ERROR: Base.Meta.ParseError("invalid syntax \"2.^\"; add space(s) to clarify")
Stacktrace:
 [1] #parse#1(::Bool, ::Bool, ::Bool, ::typeof(Base.Meta.parse), ::String, ::Int64) at ./meta.jl:184
 [2] #parse at ./none:0 [inlined]
 [3] #parse#4(::Bool, ::Bool, ::typeof(Base.Meta.parse), ::String) at ./meta.jl:215
 [4] parse(::String) at ./meta.jl:215
 [5] top-level scope at REPL[11]:1

```

**Comment by ZacLN** [comment]
fixed by: https://github.com/JuliaLang/Tokenize.jl/commit/228f0ad17fc079511d1de863dd55294cbf771f03 . Will need to tag and require new Tokenize release

**Comment by KristofferC** [comment]
New version should be registered soon.

**Comment by ZacLN** [comment]
Ta, this is fixed in the (pending) v1.1.0 release
